arangod/Agency/State.h

```diff
@@ -238,7 +238,7 @@ class State {
   /// @brief Check collections
   bool checkCollections();
 
-  /// @brief Check collection sanity
+  /// @brief Check collection existence
```

Comment:
/// @brief Check collection integrity

Reply:
No, it definitely checks for the _existence_ of a collection